tools/ide_plugins/pysa-vscode/package.json

```diff
@@ -38,5 +38,34 @@
     "tslint-microsoft-contrib": "^5.0.1",
     "typescript": "^3.5.1",
     "vscode": "^1.1.7"
+  },
+  "categories": [
+    "Programming Languages"
+  ],
+  "contributes": {
+      "languages": [{
+          "id": "pysa",
+          "aliases": ["Pysa", "pysa"],
+          "extensions": [".pysa"],
+          "configuration": "./language-configuration.json"
+      },
+      {
+          "id": "taint",
+          "aliases": ["Pysa", "taint"],
```

Comment:
Is it OK for two languages to share the same alias? What is the alias used for?

Reply:
Honestly, I'm not 100% sure but as far as I can tell, this would change the way the language is displayed in VSCode:

![image](https://user-images.githubusercontent.com/38268331/118568912-cffd2680-b746-11eb-8c61-8c3b43e76012.png)

I did some digging around and it seems it might possibly cause problems like the one mentioned [here](https://github.com/microsoft/vscode-docs/issues/2862), so I just changed the aliases to be `Pysa Model` and `Pysa Config`